gatsby/content/blog/2022/07/2022-08-03-synapse-1.64.mdx

```diff
@@ -0,0 +1,98 @@
+---
+date: '2022-08-03'
+title: Synapse 1.64 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+It's that time again: there's a new Synapse release, fresh out of the oven!
+Let's take a look at what's inside [Synapse
+1.64](https://github.com/matrix-org/synapse/releases/tag/v1.64.0).
+
+## Delegating email verification is now deprecated
+
+[Synapse 1.4.0](https://github.com/matrix-org/synapse/releases/tag/v1.4.0)
+introduced a configuration option (`account_threepid_delegates.email`) to allow
+homeservers to delegate validating the ownership of email addresses to an
+external identity server. This validation is used by Synapse when associating
+email addresses to a Matrix account, or before performing a password reset.
+
+As of Synapse 1.64, this option is deprecated. This is because this option
+relies on old API endpoints that have since been removed from the Matrix
+specification. Server administrators who still need the ability to validate the
```

Comment:
Can we cross link to the bit of the spec which says that the endpoints are deprecated?

Reply:
I can do some spelunking but the latest version of the spec doesn't even mention the v1 IS endpoints.

Reply:
https://spec.matrix.org/v1.1/changelog/#identity-service-api and 	https://github.com/matrix-org/matrix-spec-proposals/pull/2713 are probably what you want